remix-lib/src/execution/txFormat.js

```diff
@@ -413,6 +417,10 @@ module.exports = {
           } else if (params.charAt(j) === ']') {  // // Decrease count if an array closing bracket is received (To handle nested array)
             bracketCount--
           }
+          // Throw error if end of params string is arrived but couldn't get end of tuple
+          if (bracketCount !== 0 && j === params.length - 1) {
+            throw new Error('invalid tuple params')
```

Comment:
can we give here more info or this is too generic?

Reply:
Lets keep it generic for now.